### PR TITLE
*: Support builtin function round(X), round(X, D)

### PR DESCRIPTION
**Pull request by shenli**
@coocood @zimulala @zxylvlp 
Fix https://github.com/pingcap/tidb/issues/1207

### CODE REVIEW

File: evaluator/builtin_math.go [hunk 1]
```diff
@@ -81,3 +81,24 @@ func builtinPow(args []types.Datum, _ context.Context) (d types.Datum, err error
 	d.SetFloat64(math.Pow(x, y))
 	return d, nil
 }
+
+// See: http://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_round
+func builtinRound(args []types.Datum, _ context.Context) (d types.Datum, err error) {
+	x, err := args[0].ToFloat64()
+	if err != nil {
+		d.SetNull()
```
Comment (zimulala) [inline evaluator/builtin_math.go]: This line can be removed ？

**Comment by shenli** [comment]
PTAL @zimulala

File: util/types/helper.go [hunk 2]
```diff
@@ -32,6 +32,18 @@ func RoundFloat(f float64) float64 {
 	return math.Trunc(f + math.Copysign(0.5, f))
 }
 
+// Round rounds the argument X to D decimal places.
```
Comment (zxylvlp) [inline util/types/helper.go]: X->f and D->dec?

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
PTAL @zxylvlp

**Comment by zxylvlp** [comment]
LGTM

**Pull request merged**